addon/components/o-s-s/copy.ts

```diff
@@ -1,16 +1,19 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 
 import ToastService from '@upfluence/oss-components/services/toast';
 
 interface OSSCopyArgs {
   value: string;
+  inline?: boolean;
 }
 
 export default class OSSCopy extends Component<OSSCopyArgs> {
   @service intl: any;
   @service declare toast: ToastService;
+  @tracked inline: boolean = this.args.inline || false;
```

Comment:
Nit: use `??` instead of `||` because you want to set `false` for undefined value only